libs/components/src/lib/alert/alert.scss

```
@@ -83,8 +79,8 @@ $alert-transition-delay: --transition-delay;
 	display: flex;
 	align-items: center;
 	padding: 16px;
-	background-color: var(--vvd-color-neutral-50);
 	border-radius: #{border-radius-variables.$border-radius-normal};
+	background-color: var(--vvd-color-neutral-50);
```

```suggestion
	background-color: var(#{constants.$vvd-color-neutral-50});
```
not related to the PR but better like this :)